### ISSUE TITLE
possible GDM bug?

### ISSUE THREAD

**Issue by sevachka** [nana-4/Flat-Plat#22]
I had a strange behaviour, after I did install the GDM-login-screen: It could not pass my password! I did enter the password and nothing happend. I had to switch to TTY2 and back in order to type the password again and to be able to login. 
Same problem within the gnome-shell when I had to enter my password for pamac-updates/installations.
After removing the GDM-part, it worked again.

**Comment by bugaevc** [comment]
I noticed the same behavior too today, and was able to reproduce it several times in a row, spanning across reboots.

**Comment by bugaevc** [comment]
It's not actually related to GDM and ttys. I was having the same issue with systemd-ask-password - it would hang until the parent process (systemctl in my case) times out waiting.

**Comment by sevachka** [comment]
true, same problem here. But still: after removing GDM (restoring backup) it again worked system-wide.

**Comment by bugaevc** [comment]
Yeah, restoring the file fixed it for me too.

**Comment by nana-4** [comment]
Sorry for the late reply and the inconvenience.
Maybe this problem should have been fixed with the added commit, but could you please test it just in case?

**Comment by bugaevc** [comment]
Works fine now, thanks!

**Comment by sevachka** [comment]
No problems either, yet.

**Comment by zagortenay333** [comment]
This is interesting. I  have a script which generates a `gresource` file: https://github.com/zagortenay333/ciliora-tertia-shell/blob/427287e76342b4fcefaca21e9aee30f79157f129/Ciliora-Tertia/gnome-shell/gresource-gen#L23-L29

I noticed that when the `gresource.xml` file is not in the same dir as the assets I get this same issue. I even reported it against gdm: https://bugzilla.gnome.org/show_bug.cgi?id=762791

How did you figure that only these 3 assets `(process-working, no-events and no-notifications)` the culprit?

**Comment by nana-4** [comment]
@zagortenay333 
Previously, there was a problem that gnome-shell has been clashed when 3.18 theme apply to 3.16.
At that time, I've noticed that the bloated(?) `process-working` is due.
Therefore, I've imagined that it's also the culprit this time.

`no-events` and `no-notifications` are just because they are not displayed when they aren't in parent directory.
See lines 1040-1041: https://github.com/GNOME/gnome-shell/blob/master/js/ui/calendar.js

(Sorry for my bad English.)
